NCEP/gdas_utility.py

```diff
@@ -221,6 +227,13 @@ def process_data(self):
         ds['geopotential_at_surface'] = ds['geopotential_at_surface'].squeeze('batch')
         ds['land_sea_mask'] = ds['land_sea_mask'].squeeze('batch')
 
+        # Update geopotential unit to m2/s2 by multiplying 9.80665
+        ds['geopotential_at_surface'] = ds['geopotential_at_surface'] * 9.80665
+        ds['geopotential'] = ds['geopotential'] * 9.80665
+
+        # Update total_precipitation_6hr unit to (m) from (kg/m^2) by dividing it by 1000kg/m³×9.81m/s²
+        ds['total_precipitation_6hr'] = ds['total_precipitation_6hr'] / 9806.65
```

Comment:
@SadeghTabas-NOAA Can you double check the unit here? I t looks to me we only need:  APCP unit kg/m^2 / water_density pho=1000 kg/m3) to get unit (m), why do we have 9.81m/s here?

Reply:
this is the one we get from ERA5 https://codes.ecmwf.int/grib/param-db/?id=212

Reply:
@junwang-noaa if there is nothing on gdas to use instead of toa_incident_solar_radiation, we can calculate Extraterrestrial Radiation using https://github.com/pingswept/pysolar and using it.